crates/ruma-identity-service-api/src/lookup/lookup_3pid/v2.rs

```diff
@@ -1,4 +1,4 @@
-//! [POST /_matrix/identity/v2/lookup](https://matrix.org/docs/spec/identity_service/0.3.0#post-matrix-identity-v2-lookup)
+//! [POST /_matrix/identity/v2/lookup](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-lookup)
```

Comment:
This is an actual link *fix*, but the changes from the previous commit aren't. Can you update that commit messages to say "Update … links"?